modules/ebi-metagenomics/sanntis/tests/main.nf.test

```diff
@@ -0,0 +1,35 @@
+nextflow_process {
+
+    name "Test Process SANNTIS"
+    script "../main.nf"
+    process "SANNTIS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "sanntis"
+
+    test("test_sanntis") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'MGYG000300113' ], // meta map
+                    file("${moduleDir}/tests/fixtures/IPS.subset.tsv", checkIfExists: true),
+                    file("${moduleDir}/tests/fixtures/MGYG000300113.gbk", checkIfExists: true)
+                    ]
+
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
```

Comment:
add a stub test please, check https://github.com/nf-core/modules/blob/master/modules/nf-core/goat/taxonsearch/tests/main.nf.test last test

Reply:
Added in https://github.com/EBI-Metagenomics/nf-modules/commit/bbec9cfd033abab09713d17248ee60e697d91da0, hope that's how it's supposed to be done.